Automattic/jetpack
# Assistant: Make call-to-actions point to checkout page

#### Changes proposed in this Pull Request:

This PR makes sure the call-to-actions in the Jetpack plugin Assistant link to the checkout page, instead of the pricing page.

#### Jetpack product discussion


Context: D76018-code

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






- Spin up a new JN site with the live branch feature, or use your test site
- Visit the Assistant at `/wp-admin/admin.php?page=jetpack#/recommendations`
- Go through the complete flow
- Check that the CTA in the product cards link to the checkout page, with the product in cart

#### Captures

**Product suggestion page**
<img width="860" alt="Screen Shot 2022-03-24 at 11 48 21 AM" src="https://user-images.githubusercontent.com/1620183/159957260-469064be-848b-4134-9f26-4781ee726b25.png">

**Summary page**
<img width="375" alt="Screen Shot 2022-03-24 at 11 50 18 AM" src="https://user-images.githubusercontent.com/1620183/159957264-48abe564-2527-4576-bcc4-8d031ffe10dc.png">
